### DISCUSSION TITLE
TMA RTC 관련 질문

### DISCUSSION THREAD

**Discussion by ghost** [luftaquila/monolith#29] [Q&A] [answered]
안녕하세요, 병준님!
국민대학교 kust입니다.
병준님께서 공유해주신 TMA를 성공적으로 사용하고 있습니다! 먼저 정말 감사드린다는 말씀 드리고 싶네요.

다름이 아니고, TMA에 RTC에 관해서 궁금한 점이 있어서 질문을 남깁니다.
monolith 깃허브 문서대로 ESP32까지 사용하여 RTC 기능을 활성화하였습니다.

![image](https://github.com/user-attachments/assets/64323b08-7f9a-46a8-96c4-242f4c5a2410)

사진에서 볼 수 있듯이, RTC 날짜까지는 SD로그명에 성공적으로 반영이 되어있습니다만,
시간부터는 실제 시간과 다른 시간으로 나와있네요.
혹시 이런 문제점을 겪으신 적이 있는지 궁금해서 연락드립니다.
원 코드를 바꾸거나 이러진 않았는데, rtc 전체가 아니라 시간만 달라서 rtc 기능 자체가 잘못 설정되어 있지는 않은 것 같은데, esp32로 웹 시간을 가져올 때 애초에 잘못된 시간을 가져와서 이런 문제가 생길 수 있는지 궁금합니다!!

바쁘실텐데 미리 감사드립니다.

**Comment by luftaquila** [comment]
안녕하세요.

> 시간부터는 실제 시간과 다른 시간으로 나와있네요.

에서 `시간` 이 정확히 어떤 시간일까요?

**Reply by ghost** [reply, marked as answer]
아.. 혼자 생각하다가 생각해보니 서버 시간이 한국이 아니라 한국보다 9시간 늦는 utc(Coordinated Universal Time)에 맞춰서 있어서 생긴 문제네요...ㅎㅎ
해결했습니다...!

**Reply by luftaquila** [reply]
아하 네 ㅎㅎ 서버 로컬 타임존 변경하시면 되겠네요. 화이팅입니다!
